devito/types/sparse.py

```diff
@@ -669,8 +675,11 @@ def interpolate(self, expr, offset=0, increment=False, self_subs={}):
 
```

Comment:
Since `.interpolate` is changing, I expect to see new tests in `test_interpolation.py`. But I see none.

Reply:
It's not changing it's just following the changes in staggered but this is all already tested